### PR TITLE
Less unicode error logging

### PR DESCRIPTION
**Pull request by meanrin**
Fix for #43 

Change `UnicodeError` related logging to info

### CODE REVIEW

**Review by Dmitriy-NK — APPROVED** [review]
LGTM

File: credsweeper/utils/util.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def read_file(cls, path: str, encodings: List[str] = default_encodings) -> List[
                     diff_data = file.read().split("\n")
                 break
             except UnicodeError:
-                logging.warning(f"UnicodeError: Can't read content from \"{path}\" as {encoding}.")
+                logging.info(f"UnicodeError: Can't read content from \"{path}\" as {encoding}.")
             except Exception as exc:
                 logging.error(f"Unexpected Error: Can't read \"{path}\" as {encoding}. Error message: {exc}")
```
Comment (ARKAD97) [inline credsweeper/utils/util.py:100]: May be we should add warning only if none of the encodings matches.
And, maybe, add info with specific encoding.

TL; DR: I think we should log once per file, not mutliple logs per every encoding

**Review by ARKAD97 — APPROVED** [review]
LGTM
